web_source/travellers/Michelle_H.qmd

```diff
@@ -0,0 +1,31 @@
+---
+title: "Michelle H"
+subtitle: "World Traveller"
+image: ![World Traveller](https://media.self.com/photos/5f0885ffef7a10ffa6640daa/4:3/w_5240,h_3929,c_limit/travel_plane_corona.jpeg)
```

Comment:
Great profile! I think this line doesn't need the normal markdown formatting, it should just be the link without the ![xxx] or brackets around the link.

Reply:
Thanks a lot Emma. Good spot! I've now made this change.